<?xml version="1.0" encoding="UTF-8"?>
<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Strict//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-strict.dtd">
<html xmlns="http://www.w3.org/1999/xhtml">
<head>
    <title>student_client</title>
    <meta name="generator" content="HTMLTestRunner 0.8.3"/>
    <meta http-equiv="Content-Type" content="text/html; charset=UTF-8"/>
    
<style type="text/css" media="screen">
body        { font-family: verdana, arial, helvetica, sans-serif; font-size: 80%; }
table       { font-size: 100%; }
pre         { }

/* -- heading ---------------------------------------------------------------------- */
h1 {
	font-size: 16pt;
	color: gray;
}
.heading {
    margin-top: 0ex;
    margin-bottom: 1ex;
}

.heading .attribute {
    margin-top: 1ex;
    margin-bottom: 0;
}

.heading .description {
    margin-top: 4ex;
    margin-bottom: 6ex;
}

/* -- css div popup ------------------------------------------------------------------------ */
a.popup_link {
}

a.popup_link:hover {
    color: red;
}

.popup_window {
    display: none;
    position: relative;
    left: 0px;
    top: 0px;
    /*border: solid #627173 1px; */
    padding: 10px;
    background-color: #E6E6D6;
    font-family: "Lucida Console", "Courier New", Courier, monospace;
    text-align: left;
    font-size: 8pt;
    width: 500px;
}

}
/* -- report ------------------------------------------------------------------------ */
#show_detail_line {
    margin-top: 3ex;
    margin-bottom: 1ex;
}
#result_table {
    width: 80%;
    border-collapse: collapse;
    border: 1px solid #777;
}
#header_row {
    font-weight: bold;
    color: white;
    background-color: #777;
}
#result_table td {
    border: 1px solid #777;
    padding: 2px;
}
#total_row  { font-weight: bold; }
.passClass  { background-color: #6c6; }
.failClass  { background-color: #c60; }
.errorClass { background-color: #c00; }
.passCase   { color: #6c6; }
.failCase   { color: #c60; font-weight: bold; }
.errorCase  { color: #c00; font-weight: bold; }
.hiddenRow  { display: none; }
.testcase   { margin-left: 2em; }


/* -- ending ---------------------------------------------------------------------- */
#ending {
}

</style>

</head>
<body>
<script language="javascript" type="text/javascript"><!--
output_list = Array();

/* level - 0:Summary; 1:Failed; 2:All */
function showCase(level) {
    trs = document.getElementsByTagName("tr");
    for (var i = 0; i < trs.length; i++) {
        tr = trs[i];
        id = tr.id;
        if (id.substr(0,2) == 'ft') {
            if (level < 1) {
                tr.className = 'hiddenRow';
            }
            else {
                tr.className = '';
            }
        }
        if (id.substr(0,2) == 'pt') {
            if (level > 1) {
                tr.className = '';
            }
            else {
                tr.className = 'hiddenRow';
            }
        }
    }
}


function showClassDetail(cid, count) {
    var id_list = Array(count);
    var toHide = 1;
    for (var i = 0; i < count; i++) {
        tid0 = 't' + cid.substr(1) + '.' + (i+1);
        tid = 'f' + tid0;
        tr = document.getElementById(tid);
        if (!tr) {
            tid = 'p' + tid0;
            tr = document.getElementById(tid);
        }
        id_list[i] = tid;
        if (tr.className) {
            toHide = 0;
        }
    }
    for (var i = 0; i < count; i++) {
        tid = id_list[i];
        if (toHide) {
            document.getElementById('div_'+tid).style.display = 'none'
            document.getElementById(tid).className = 'hiddenRow';
        }
        else {
            document.getElementById(tid).className = '';
        }
    }
}


function showTestDetail(div_id){
    var details_div = document.getElementById(div_id)
    var displayState = details_div.style.display
    // alert(displayState)
    if (displayState != 'block' ) {
        displayState = 'block'
        details_div.style.display = 'block'
    }
    else {
        details_div.style.display = 'none'
    }
}


function html_escape(s) {
    s = s.replace(/&/g,'&amp;');
    s = s.replace(/</g,'&lt;');
    s = s.replace(/>/g,'&gt;');
    return s;
}

/* obsoleted by detail in <div>
function showOutput(id, name) {
    var w = window.open("", //url
                    name,
                    "resizable,scrollbars,status,width=800,height=450");
    d = w.document;
    d.write("<pre>");
    d.write(html_escape(output_list[id]));
    d.write("\n");
    d.write("<a href='javascript:window.close()'>close</a>\n");
    d.write("</pre>\n");
    d.close();
}
*/
--></script>

<div class='heading'>
<h1>student_client</h1>
<p class='attribute'><strong>Start Time:</strong> 2019-07-08 17:38:44</p>
<p class='attribute'><strong>Duration:</strong> 0:03:04.174094</p>
<p class='attribute'><strong>Status:</strong> Pass 4 Error 5</p>

<p class='description'>测试用例执行结果</p>
</div>



<p id='show_detail_line'>Show
<a href='javascript:showCase(0)'>Summary</a>
<a href='javascript:showCase(1)'>Failed</a>
<a href='javascript:showCase(2)'>All</a>
</p>
<table id='result_table'>
<colgroup>
<col align='left' />
<col align='right' />
<col align='right' />
<col align='right' />
<col align='right' />
<col align='right' />
</colgroup>
<tr id='header_row'>
    <td>Test Group/Test case</td>
    <td>Count</td>
    <td>Pass</td>
    <td>Fail</td>
    <td>Error</td>
    <td>View</td>
</tr>

<tr class='errorClass'>
    <td>testIOS.test_student_client.AppTests</td>
    <td>9</td>
    <td>4</td>
    <td>0</td>
    <td>5</td>
    <td><a href="javascript:showClassDetail('c1',9)">Detail</a></td>
</tr>

<tr id='pt1.1' class='hiddenRow'>
    <td class='none'><div class='testcase'>test01_student_login</div></td>
    <td colspan='5' align='center'>

    <!--css div popup start-->
    <a class="popup_link" onfocus='this.blur();' href="javascript:showTestDetail('div_pt1.1')" >
        pass</a>

    <div id='div_pt1.1' class="popup_window">
        <div style='text-align: right; color:red;cursor:pointer'>
        <a onfocus='this.blur();' onclick="document.getElementById('div_pt1.1').style.display = 'none' " >
           [x]</a>
        </div>
        <pre>
        
pt1.1: 

        </pre>
    </div>
    <!--css div popup end-->

    </td>
</tr>

<tr id='pt1.2' class='hiddenRow'>
    <td class='none'><div class='testcase'>test02_student_info</div></td>
    <td colspan='5' align='center'>

    <!--css div popup start-->
    <a class="popup_link" onfocus='this.blur();' href="javascript:showTestDetail('div_pt1.2')" >
        pass</a>

    <div id='div_pt1.2' class="popup_window">
        <div style='text-align: right; color:red;cursor:pointer'>
        <a onfocus='this.blur();' onclick="document.getElementById('div_pt1.2').style.display = 'none' " >
           [x]</a>
        </div>
        <pre>
        
pt1.2: 

        </pre>
    </div>
    <!--css div popup end-->

    </td>
</tr>

<tr id='pt1.3' class='hiddenRow'>
    <td class='none'><div class='testcase'>test03_enter_my_course</div></td>
    <td colspan='5' align='center'>

    <!--css div popup start-->
    <a class="popup_link" onfocus='this.blur();' href="javascript:showTestDetail('div_pt1.3')" >
        pass</a>

    <div id='div_pt1.3' class="popup_window">
        <div style='text-align: right; color:red;cursor:pointer'>
        <a onfocus='this.blur();' onclick="document.getElementById('div_pt1.3').style.display = 'none' " >
           [x]</a>
        </div>
        <pre>
        
pt1.3: 

        </pre>
    </div>
    <!--css div popup end-->

    </td>
</tr>

<tr id='pt1.4' class='hiddenRow'>
    <td class='none'><div class='testcase'>test04_enter_teacher_live_broadcast</div></td>
    <td colspan='5' align='center'>

    <!--css div popup start-->
    <a class="popup_link" onfocus='this.blur();' href="javascript:showTestDetail('div_pt1.4')" >
        pass</a>

    <div id='div_pt1.4' class="popup_window">
        <div style='text-align: right; color:red;cursor:pointer'>
        <a onfocus='this.blur();' onclick="document.getElementById('div_pt1.4').style.display = 'none' " >
           [x]</a>
        </div>
        <pre>
        
pt1.4: 

        </pre>
    </div>
    <!--css div popup end-->

    </td>
</tr>

<tr id='ft1.5' class='none'>
    <td class='errorCase'><div class='testcase'>test05_enter_open_class</div></td>
    <td colspan='5' align='center'>

    <!--css div popup start-->
    <a class="popup_link" onfocus='this.blur();' href="javascript:showTestDetail('div_ft1.5')" >
        error</a>

    <div id='div_ft1.5' class="popup_window">
        <div style='text-align: right; color:red;cursor:pointer'>
        <a onfocus='this.blur();' onclick="document.getElementById('div_ft1.5').style.display = 'none' " >
           [x]</a>
        </div>
        <pre>
        
ft1.5: Traceback (most recent call last):
  File "/Users/jason.lik/DaDa_project/abc-qa-app-UITest/common/operate_element.py", line 186, in main_operate
    driver.implicitly_wait(20)
  File "/Users/jason.lik/DaDa_project/abc-qa-app-UITest/venv/lib/python3.6/site-packages/selenium/webdriver/remote/webdriver.py", line 912, in implicitly_wait
    'implicit': int(float(time_to_wait) * 1000)})
  File "/Users/jason.lik/DaDa_project/abc-qa-app-UITest/venv/lib/python3.6/site-packages/selenium/webdriver/remote/webdriver.py", line 321, in execute
    self.error_handler.check_response(response)
  File "/Users/jason.lik/DaDa_project/abc-qa-app-UITest/venv/lib/python3.6/site-packages/appium/webdriver/errorhandler.py", line 29, in check_response
    raise wde
  File "/Users/jason.lik/DaDa_project/abc-qa-app-UITest/venv/lib/python3.6/site-packages/appium/webdriver/errorhandler.py", line 24, in check_response
    super(MobileErrorHandler, self).check_response(response)
  File "/Users/jason.lik/DaDa_project/abc-qa-app-UITest/venv/lib/python3.6/site-packages/selenium/webdriver/remote/errorhandler.py", line 242, in check_response
    raise exception_class(message, screen, stacktrace)
selenium.common.exceptions.InvalidSessionIdException: Message: A session is either terminated or not started


During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/Users/jason.lik/DaDa_project/abc-qa-app-UITest/common/operate_element.py", line 152, in wrapper
    result = function(*a, **w)
  File "/Users/jason.lik/DaDa_project/abc-qa-app-UITest/testIOS/test_student_client.py", line 49, in test05_enter_open_class
    enter_open_class(self.driver)
  File "/Users/jason.lik/DaDa_project/abc-qa-app-UITest/testIOS/student_client/case/test05_open_class.py", line 17, in enter_open_class
    __case_collection(driver, 'ipad')
  File "/Users/jason.lik/DaDa_project/abc-qa-app-UITest/testIOS/student_client/case/test05_open_class.py", line 12, in __case_collection
    main_operate(driver, platform, **open_class)
  File "/Users/jason.lik/DaDa_project/abc-qa-app-UITest/common/operate_element.py", line 207, in main_operate
    yaml + platform + '/error_screenshot/') + '/' + get_current_hour_minute() + '_error.png')
  File "/Users/jason.lik/DaDa_project/abc-qa-app-UITest/common/operate_element.py", line 112, in __error_screenshot
    driver.get_screenshot_as_file(screenshot_path)
  File "/Users/jason.lik/DaDa_project/abc-qa-app-UITest/venv/lib/python3.6/site-packages/selenium/webdriver/remote/webdriver.py", line 1032, in get_screenshot_as_file
    png = self.get_screenshot_as_png()
  File "/Users/jason.lik/DaDa_project/abc-qa-app-UITest/venv/lib/python3.6/site-packages/selenium/webdriver/remote/webdriver.py", line 1064, in get_screenshot_as_png
    return base64.b64decode(self.get_screenshot_as_base64().encode('ascii'))
  File "/Users/jason.lik/DaDa_project/abc-qa-app-UITest/venv/lib/python3.6/site-packages/selenium/webdriver/remote/webdriver.py", line 1074, in get_screenshot_as_base64
    return self.execute(Command.SCREENSHOT)['value']
  File "/Users/jason.lik/DaDa_project/abc-qa-app-UITest/venv/lib/python3.6/site-packages/selenium/webdriver/remote/webdriver.py", line 321, in execute
    self.error_handler.check_response(response)
  File "/Users/jason.lik/DaDa_project/abc-qa-app-UITest/venv/lib/python3.6/site-packages/appium/webdriver/errorhandler.py", line 29, in check_response
    raise wde
  File "/Users/jason.lik/DaDa_project/abc-qa-app-UITest/venv/lib/python3.6/site-packages/appium/webdriver/errorhandler.py", line 24, in check_response
    super(MobileErrorHandler, self).check_response(response)
  File "/Users/jason.lik/DaDa_project/abc-qa-app-UITest/venv/lib/python3.6/site-packages/selenium/webdriver/remote/errorhandler.py", line 242, in check_response
    raise exception_class(message, screen, stacktrace)
selenium.common.exceptions.InvalidSessionIdException: Message: A session is either terminated or not started


During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/Users/jason.lik/DaDa_project/abc-qa-app-UITest/common/operate_element.py", line 158, in wrapper
    t(*a)
TypeError: 'classmethod' object is not callable
Traceback (most recent call last):
  File "/Users/jason.lik/DaDa_project/abc-qa-app-UITest/common/operate_element.py", line 186, in main_operate
    driver.implicitly_wait(20)
  File "/Users/jason.lik/DaDa_project/abc-qa-app-UITest/venv/lib/python3.6/site-packages/selenium/webdriver/remote/webdriver.py", line 912, in implicitly_wait
    'implicit': int(float(time_to_wait) * 1000)})
  File "/Users/jason.lik/DaDa_project/abc-qa-app-UITest/venv/lib/python3.6/site-packages/selenium/webdriver/remote/webdriver.py", line 321, in execute
    self.error_handler.check_response(response)
  File "/Users/jason.lik/DaDa_project/abc-qa-app-UITest/venv/lib/python3.6/site-packages/appium/webdriver/errorhandler.py", line 29, in check_response
    raise wde
  File "/Users/jason.lik/DaDa_project/abc-qa-app-UITest/venv/lib/python3.6/site-packages/appium/webdriver/errorhandler.py", line 24, in check_response
    super(MobileErrorHandler, self).check_response(response)
  File "/Users/jason.lik/DaDa_project/abc-qa-app-UITest/venv/lib/python3.6/site-packages/selenium/webdriver/remote/errorhandler.py", line 242, in check_response
    raise exception_class(message, screen, stacktrace)
selenium.common.exceptions.InvalidSessionIdException: Message: A session is either terminated or not started


During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/Users/jason.lik/DaDa_project/abc-qa-app-UITest/common/operate_element.py", line 152, in wrapper
    result = function(*a, **w)
  File "/Users/jason.lik/DaDa_project/abc-qa-app-UITest/testIOS/test_student_client.py", line 49, in test05_enter_open_class
    enter_open_class(self.driver)
  File "/Users/jason.lik/DaDa_project/abc-qa-app-UITest/testIOS/student_client/case/test05_open_class.py", line 17, in enter_open_class
    __case_collection(driver, 'ipad')
  File "/Users/jason.lik/DaDa_project/abc-qa-app-UITest/testIOS/student_client/case/test05_open_class.py", line 12, in __case_collection
    main_operate(driver, platform, **open_class)
  File "/Users/jason.lik/DaDa_project/abc-qa-app-UITest/common/operate_element.py", line 207, in main_operate
    yaml + platform + '/error_screenshot/') + '/' + get_current_hour_minute() + '_error.png')
  File "/Users/jason.lik/DaDa_project/abc-qa-app-UITest/common/operate_element.py", line 112, in __error_screenshot
    driver.get_screenshot_as_file(screenshot_path)
  File "/Users/jason.lik/DaDa_project/abc-qa-app-UITest/venv/lib/python3.6/site-packages/selenium/webdriver/remote/webdriver.py", line 1032, in get_screenshot_as_file
    png = self.get_screenshot_as_png()
  File "/Users/jason.lik/DaDa_project/abc-qa-app-UITest/venv/lib/python3.6/site-packages/selenium/webdriver/remote/webdriver.py", line 1064, in get_screenshot_as_png
    return base64.b64decode(self.get_screenshot_as_base64().encode('ascii'))
  File "/Users/jason.lik/DaDa_project/abc-qa-app-UITest/venv/lib/python3.6/site-packages/selenium/webdriver/remote/webdriver.py", line 1074, in get_screenshot_as_base64
    return self.execute(Command.SCREENSHOT)['value']
  File "/Users/jason.lik/DaDa_project/abc-qa-app-UITest/venv/lib/python3.6/site-packages/selenium/webdriver/remote/webdriver.py", line 321, in execute
    self.error_handler.check_response(response)
  File "/Users/jason.lik/DaDa_project/abc-qa-app-UITest/venv/lib/python3.6/site-packages/appium/webdriver/errorhandler.py", line 29, in check_response
    raise wde
  File "/Users/jason.lik/DaDa_project/abc-qa-app-UITest/venv/lib/python3.6/site-packages/appium/webdriver/errorhandler.py", line 24, in check_response
    super(MobileErrorHandler, self).check_response(response)
  File "/Users/jason.lik/DaDa_project/abc-qa-app-UITest/venv/lib/python3.6/site-packages/selenium/webdriver/remote/errorhandler.py", line 242, in check_response
    raise exception_class(message, screen, stacktrace)
selenium.common.exceptions.InvalidSessionIdException: Message: A session is either terminated or not started


During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/Users/jason.lik/DaDa_project/abc-qa-app-UITest/common/operate_element.py", line 158, in wrapper
    t(*a)
TypeError: 'classmethod' object is not callable


        </pre>
    </div>
    <!--css div popup end-->

    </td>
</tr>

<tr id='ft1.6' class='none'>
    <td class='errorCase'><div class='testcase'>test06_enter_after_class_tasks</div></td>
    <td colspan='5' align='center'>

    <!--css div popup start-->
    <a class="popup_link" onfocus='this.blur();' href="javascript:showTestDetail('div_ft1.6')" >
        error</a>

    <div id='div_ft1.6' class="popup_window">
        <div style='text-align: right; color:red;cursor:pointer'>
        <a onfocus='this.blur();' onclick="document.getElementById('div_ft1.6').style.display = 'none' " >
           [x]</a>
        </div>
        <pre>
        
ft1.6: Traceback (most recent call last):
  File "/Users/jason.lik/DaDa_project/abc-qa-app-UITest/common/operate_element.py", line 186, in main_operate
    driver.implicitly_wait(20)
  File "/Users/jason.lik/DaDa_project/abc-qa-app-UITest/venv/lib/python3.6/site-packages/selenium/webdriver/remote/webdriver.py", line 912, in implicitly_wait
    'implicit': int(float(time_to_wait) * 1000)})
  File "/Users/jason.lik/DaDa_project/abc-qa-app-UITest/venv/lib/python3.6/site-packages/selenium/webdriver/remote/webdriver.py", line 321, in execute
    self.error_handler.check_response(response)
  File "/Users/jason.lik/DaDa_project/abc-qa-app-UITest/venv/lib/python3.6/site-packages/appium/webdriver/errorhandler.py", line 29, in check_response
    raise wde
  File "/Users/jason.lik/DaDa_project/abc-qa-app-UITest/venv/lib/python3.6/site-packages/appium/webdriver/errorhandler.py", line 24, in check_response
    super(MobileErrorHandler, self).check_response(response)
  File "/Users/jason.lik/DaDa_project/abc-qa-app-UITest/venv/lib/python3.6/site-packages/selenium/webdriver/remote/errorhandler.py", line 242, in check_response
    raise exception_class(message, screen, stacktrace)
selenium.common.exceptions.InvalidSessionIdException: Message: A session is either terminated or not started


During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/Users/jason.lik/DaDa_project/abc-qa-app-UITest/common/operate_element.py", line 152, in wrapper
    result = function(*a, **w)
  File "/Users/jason.lik/DaDa_project/abc-qa-app-UITest/testIOS/test_student_client.py", line 53, in test06_enter_after_class_tasks
    enter_after_class_tasks(self.driver)
  File "/Users/jason.lik/DaDa_project/abc-qa-app-UITest/testIOS/student_client/case/test06_after_class_tasks.py", line 17, in enter_after_class_tasks
    __case_collection(driver, 'ipad')
  File "/Users/jason.lik/DaDa_project/abc-qa-app-UITest/testIOS/student_client/case/test06_after_class_tasks.py", line 12, in __case_collection
    main_operate(driver, platform, **after_class_tasks)
  File "/Users/jason.lik/DaDa_project/abc-qa-app-UITest/common/operate_element.py", line 207, in main_operate
    yaml + platform + '/error_screenshot/') + '/' + get_current_hour_minute() + '_error.png')
  File "/Users/jason.lik/DaDa_project/abc-qa-app-UITest/common/operate_element.py", line 112, in __error_screenshot
    driver.get_screenshot_as_file(screenshot_path)
  File "/Users/jason.lik/DaDa_project/abc-qa-app-UITest/venv/lib/python3.6/site-packages/selenium/webdriver/remote/webdriver.py", line 1032, in get_screenshot_as_file
    png = self.get_screenshot_as_png()
  File "/Users/jason.lik/DaDa_project/abc-qa-app-UITest/venv/lib/python3.6/site-packages/selenium/webdriver/remote/webdriver.py", line 1064, in get_screenshot_as_png
    return base64.b64decode(self.get_screenshot_as_base64().encode('ascii'))
  File "/Users/jason.lik/DaDa_project/abc-qa-app-UITest/venv/lib/python3.6/site-packages/selenium/webdriver/remote/webdriver.py", line 1074, in get_screenshot_as_base64
    return self.execute(Command.SCREENSHOT)['value']
  File "/Users/jason.lik/DaDa_project/abc-qa-app-UITest/venv/lib/python3.6/site-packages/selenium/webdriver/remote/webdriver.py", line 321, in execute
    self.error_handler.check_response(response)
  File "/Users/jason.lik/DaDa_project/abc-qa-app-UITest/venv/lib/python3.6/site-packages/appium/webdriver/errorhandler.py", line 29, in check_response
    raise wde
  File "/Users/jason.lik/DaDa_project/abc-qa-app-UITest/venv/lib/python3.6/site-packages/appium/webdriver/errorhandler.py", line 24, in check_response
    super(MobileErrorHandler, self).check_response(response)
  File "/Users/jason.lik/DaDa_project/abc-qa-app-UITest/venv/lib/python3.6/site-packages/selenium/webdriver/remote/errorhandler.py", line 242, in check_response
    raise exception_class(message, screen, stacktrace)
selenium.common.exceptions.InvalidSessionIdException: Message: A session is either terminated or not started


During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/Users/jason.lik/DaDa_project/abc-qa-app-UITest/common/operate_element.py", line 158, in wrapper
    t(*a)
TypeError: 'classmethod' object is not callable
Traceback (most recent call last):
  File "/Users/jason.lik/DaDa_project/abc-qa-app-UITest/common/operate_element.py", line 186, in main_operate
    driver.implicitly_wait(20)
  File "/Users/jason.lik/DaDa_project/abc-qa-app-UITest/venv/lib/python3.6/site-packages/selenium/webdriver/remote/webdriver.py", line 912, in implicitly_wait
    'implicit': int(float(time_to_wait) * 1000)})
  File "/Users/jason.lik/DaDa_project/abc-qa-app-UITest/venv/lib/python3.6/site-packages/selenium/webdriver/remote/webdriver.py", line 321, in execute
    self.error_handler.check_response(response)
  File "/Users/jason.lik/DaDa_project/abc-qa-app-UITest/venv/lib/python3.6/site-packages/appium/webdriver/errorhandler.py", line 29, in check_response
    raise wde
  File "/Users/jason.lik/DaDa_project/abc-qa-app-UITest/venv/lib/python3.6/site-packages/appium/webdriver/errorhandler.py", line 24, in check_response
    super(MobileErrorHandler, self).check_response(response)
  File "/Users/jason.lik/DaDa_project/abc-qa-app-UITest/venv/lib/python3.6/site-packages/selenium/webdriver/remote/errorhandler.py", line 242, in check_response
    raise exception_class(message, screen, stacktrace)
selenium.common.exceptions.InvalidSessionIdException: Message: A session is either terminated or not started


During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/Users/jason.lik/DaDa_project/abc-qa-app-UITest/common/operate_element.py", line 152, in wrapper
    result = function(*a, **w)
  File "/Users/jason.lik/DaDa_project/abc-qa-app-UITest/testIOS/test_student_client.py", line 53, in test06_enter_after_class_tasks
    enter_after_class_tasks(self.driver)
  File "/Users/jason.lik/DaDa_project/abc-qa-app-UITest/testIOS/student_client/case/test06_after_class_tasks.py", line 17, in enter_after_class_tasks
    __case_collection(driver, 'ipad')
  File "/Users/jason.lik/DaDa_project/abc-qa-app-UITest/testIOS/student_client/case/test06_after_class_tasks.py", line 12, in __case_collection
    main_operate(driver, platform, **after_class_tasks)
  File "/Users/jason.lik/DaDa_project/abc-qa-app-UITest/common/operate_element.py", line 207, in main_operate
    yaml + platform + '/error_screenshot/') + '/' + get_current_hour_minute() + '_error.png')
  File "/Users/jason.lik/DaDa_project/abc-qa-app-UITest/common/operate_element.py", line 112, in __error_screenshot
    driver.get_screenshot_as_file(screenshot_path)
  File "/Users/jason.lik/DaDa_project/abc-qa-app-UITest/venv/lib/python3.6/site-packages/selenium/webdriver/remote/webdriver.py", line 1032, in get_screenshot_as_file
    png = self.get_screenshot_as_png()
  File "/Users/jason.lik/DaDa_project/abc-qa-app-UITest/venv/lib/python3.6/site-packages/selenium/webdriver/remote/webdriver.py", line 1064, in get_screenshot_as_png
    return base64.b64decode(self.get_screenshot_as_base64().encode('ascii'))
  File "/Users/jason.lik/DaDa_project/abc-qa-app-UITest/venv/lib/python3.6/site-packages/selenium/webdriver/remote/webdriver.py", line 1074, in get_screenshot_as_base64
    return self.execute(Command.SCREENSHOT)['value']
  File "/Users/jason.lik/DaDa_project/abc-qa-app-UITest/venv/lib/python3.6/site-packages/selenium/webdriver/remote/webdriver.py", line 321, in execute
    self.error_handler.check_response(response)
  File "/Users/jason.lik/DaDa_project/abc-qa-app-UITest/venv/lib/python3.6/site-packages/appium/webdriver/errorhandler.py", line 29, in check_response
    raise wde
  File "/Users/jason.lik/DaDa_project/abc-qa-app-UITest/venv/lib/python3.6/site-packages/appium/webdriver/errorhandler.py", line 24, in check_response
    super(MobileErrorHandler, self).check_response(response)
  File "/Users/jason.lik/DaDa_project/abc-qa-app-UITest/venv/lib/python3.6/site-packages/selenium/webdriver/remote/errorhandler.py", line 242, in check_response
    raise exception_class(message, screen, stacktrace)
selenium.common.exceptions.InvalidSessionIdException: Message: A session is either terminated or not started


During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/Users/jason.lik/DaDa_project/abc-qa-app-UITest/common/operate_element.py", line 158, in wrapper
    t(*a)
TypeError: 'classmethod' object is not callable


        </pre>
    </div>
    <!--css div popup end-->

    </td>
</tr>

<tr id='ft1.7' class='none'>
    <td class='errorCase'><div class='testcase'>test07_enter_audio_picture_book</div></td>
    <td colspan='5' align='center'>

    <!--css div popup start-->
    <a class="popup_link" onfocus='this.blur();' href="javascript:showTestDetail('div_ft1.7')" >
        error</a>

    <div id='div_ft1.7' class="popup_window">
        <div style='text-align: right; color:red;cursor:pointer'>
        <a onfocus='this.blur();' onclick="document.getElementById('div_ft1.7').style.display = 'none' " >
           [x]</a>
        </div>
        <pre>
        
ft1.7: Traceback (most recent call last):
  File "/Users/jason.lik/DaDa_project/abc-qa-app-UITest/common/operate_element.py", line 186, in main_operate
    driver.implicitly_wait(20)
  File "/Users/jason.lik/DaDa_project/abc-qa-app-UITest/venv/lib/python3.6/site-packages/selenium/webdriver/remote/webdriver.py", line 912, in implicitly_wait
    'implicit': int(float(time_to_wait) * 1000)})
  File "/Users/jason.lik/DaDa_project/abc-qa-app-UITest/venv/lib/python3.6/site-packages/selenium/webdriver/remote/webdriver.py", line 321, in execute
    self.error_handler.check_response(response)
  File "/Users/jason.lik/DaDa_project/abc-qa-app-UITest/venv/lib/python3.6/site-packages/appium/webdriver/errorhandler.py", line 29, in check_response
    raise wde
  File "/Users/jason.lik/DaDa_project/abc-qa-app-UITest/venv/lib/python3.6/site-packages/appium/webdriver/errorhandler.py", line 24, in check_response
    super(MobileErrorHandler, self).check_response(response)
  File "/Users/jason.lik/DaDa_project/abc-qa-app-UITest/venv/lib/python3.6/site-packages/selenium/webdriver/remote/errorhandler.py", line 242, in check_response
    raise exception_class(message, screen, stacktrace)
selenium.common.exceptions.InvalidSessionIdException: Message: A session is either terminated or not started


During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/Users/jason.lik/DaDa_project/abc-qa-app-UITest/common/operate_element.py", line 152, in wrapper
    result = function(*a, **w)
  File "/Users/jason.lik/DaDa_project/abc-qa-app-UITest/testIOS/test_student_client.py", line 57, in test07_enter_audio_picture_book
    enter_audio_picture_book(self.driver)
  File "/Users/jason.lik/DaDa_project/abc-qa-app-UITest/testIOS/student_client/case/test07_audio_picture_book.py", line 17, in enter_audio_picture_book
    __case_collection(driver, 'ipad')
  File "/Users/jason.lik/DaDa_project/abc-qa-app-UITest/testIOS/student_client/case/test07_audio_picture_book.py", line 12, in __case_collection
    main_operate(driver, platform, **audio_picture_book)
  File "/Users/jason.lik/DaDa_project/abc-qa-app-UITest/common/operate_element.py", line 207, in main_operate
    yaml + platform + '/error_screenshot/') + '/' + get_current_hour_minute() + '_error.png')
  File "/Users/jason.lik/DaDa_project/abc-qa-app-UITest/common/operate_element.py", line 112, in __error_screenshot
    driver.get_screenshot_as_file(screenshot_path)
  File "/Users/jason.lik/DaDa_project/abc-qa-app-UITest/venv/lib/python3.6/site-packages/selenium/webdriver/remote/webdriver.py", line 1032, in get_screenshot_as_file
    png = self.get_screenshot_as_png()
  File "/Users/jason.lik/DaDa_project/abc-qa-app-UITest/venv/lib/python3.6/site-packages/selenium/webdriver/remote/webdriver.py", line 1064, in get_screenshot_as_png
    return base64.b64decode(self.get_screenshot_as_base64().encode('ascii'))
  File "/Users/jason.lik/DaDa_project/abc-qa-app-UITest/venv/lib/python3.6/site-packages/selenium/webdriver/remote/webdriver.py", line 1074, in get_screenshot_as_base64
    return self.execute(Command.SCREENSHOT)['value']
  File "/Users/jason.lik/DaDa_project/abc-qa-app-UITest/venv/lib/python3.6/site-packages/selenium/webdriver/remote/webdriver.py", line 321, in execute
    self.error_handler.check_response(response)
  File "/Users/jason.lik/DaDa_project/abc-qa-app-UITest/venv/lib/python3.6/site-packages/appium/webdriver/errorhandler.py", line 29, in check_response
    raise wde
  File "/Users/jason.lik/DaDa_project/abc-qa-app-UITest/venv/lib/python3.6/site-packages/appium/webdriver/errorhandler.py", line 24, in check_response
    super(MobileErrorHandler, self).check_response(response)
  File "/Users/jason.lik/DaDa_project/abc-qa-app-UITest/venv/lib/python3.6/site-packages/selenium/webdriver/remote/errorhandler.py", line 242, in check_response
    raise exception_class(message, screen, stacktrace)
selenium.common.exceptions.InvalidSessionIdException: Message: A session is either terminated or not started


During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/Users/jason.lik/DaDa_project/abc-qa-app-UITest/common/operate_element.py", line 158, in wrapper
    t(*a)
TypeError: 'classmethod' object is not callable
Traceback (most recent call last):
  File "/Users/jason.lik/DaDa_project/abc-qa-app-UITest/common/operate_element.py", line 186, in main_operate
    driver.implicitly_wait(20)
  File "/Users/jason.lik/DaDa_project/abc-qa-app-UITest/venv/lib/python3.6/site-packages/selenium/webdriver/remote/webdriver.py", line 912, in implicitly_wait
    'implicit': int(float(time_to_wait) * 1000)})
  File "/Users/jason.lik/DaDa_project/abc-qa-app-UITest/venv/lib/python3.6/site-packages/selenium/webdriver/remote/webdriver.py", line 321, in execute
    self.error_handler.check_response(response)
  File "/Users/jason.lik/DaDa_project/abc-qa-app-UITest/venv/lib/python3.6/site-packages/appium/webdriver/errorhandler.py", line 29, in check_response
    raise wde
  File "/Users/jason.lik/DaDa_project/abc-qa-app-UITest/venv/lib/python3.6/site-packages/appium/webdriver/errorhandler.py", line 24, in check_response
    super(MobileErrorHandler, self).check_response(response)
  File "/Users/jason.lik/DaDa_project/abc-qa-app-UITest/venv/lib/python3.6/site-packages/selenium/webdriver/remote/errorhandler.py", line 242, in check_response
    raise exception_class(message, screen, stacktrace)
selenium.common.exceptions.InvalidSessionIdException: Message: A session is either terminated or not started


During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/Users/jason.lik/DaDa_project/abc-qa-app-UITest/common/operate_element.py", line 152, in wrapper
    result = function(*a, **w)
  File "/Users/jason.lik/DaDa_project/abc-qa-app-UITest/testIOS/test_student_client.py", line 57, in test07_enter_audio_picture_book
    enter_audio_picture_book(self.driver)
  File "/Users/jason.lik/DaDa_project/abc-qa-app-UITest/testIOS/student_client/case/test07_audio_picture_book.py", line 17, in enter_audio_picture_book
    __case_collection(driver, 'ipad')
  File "/Users/jason.lik/DaDa_project/abc-qa-app-UITest/testIOS/student_client/case/test07_audio_picture_book.py", line 12, in __case_collection
    main_operate(driver, platform, **audio_picture_book)
  File "/Users/jason.lik/DaDa_project/abc-qa-app-UITest/common/operate_element.py", line 207, in main_operate
    yaml + platform + '/error_screenshot/') + '/' + get_current_hour_minute() + '_error.png')
  File "/Users/jason.lik/DaDa_project/abc-qa-app-UITest/common/operate_element.py", line 112, in __error_screenshot
    driver.get_screenshot_as_file(screenshot_path)
  File "/Users/jason.lik/DaDa_project/abc-qa-app-UITest/venv/lib/python3.6/site-packages/selenium/webdriver/remote/webdriver.py", line 1032, in get_screenshot_as_file
    png = self.get_screenshot_as_png()
  File "/Users/jason.lik/DaDa_project/abc-qa-app-UITest/venv/lib/python3.6/site-packages/selenium/webdriver/remote/webdriver.py", line 1064, in get_screenshot_as_png
    return base64.b64decode(self.get_screenshot_as_base64().encode('ascii'))
  File "/Users/jason.lik/DaDa_project/abc-qa-app-UITest/venv/lib/python3.6/site-packages/selenium/webdriver/remote/webdriver.py", line 1074, in get_screenshot_as_base64
    return self.execute(Command.SCREENSHOT)['value']
  File "/Users/jason.lik/DaDa_project/abc-qa-app-UITest/venv/lib/python3.6/site-packages/selenium/webdriver/remote/webdriver.py", line 321, in execute
    self.error_handler.check_response(response)
  File "/Users/jason.lik/DaDa_project/abc-qa-app-UITest/venv/lib/python3.6/site-packages/appium/webdriver/errorhandler.py", line 29, in check_response
    raise wde
  File "/Users/jason.lik/DaDa_project/abc-qa-app-UITest/venv/lib/python3.6/site-packages/appium/webdriver/errorhandler.py", line 24, in check_response
    super(MobileErrorHandler, self).check_response(response)
  File "/Users/jason.lik/DaDa_project/abc-qa-app-UITest/venv/lib/python3.6/site-packages/selenium/webdriver/remote/errorhandler.py", line 242, in check_response
    raise exception_class(message, screen, stacktrace)
selenium.common.exceptions.InvalidSessionIdException: Message: A session is either terminated or not started


During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/Users/jason.lik/DaDa_project/abc-qa-app-UITest/common/operate_element.py", line 158, in wrapper
    t(*a)
TypeError: 'classmethod' object is not callable


        </pre>
    </div>
    <!--css div popup end-->

    </td>
</tr>

<tr id='ft1.8' class='none'>
    <td class='errorCase'><div class='testcase'>test08_enter_oral_practice</div></td>
    <td colspan='5' align='center'>

    <!--css div popup start-->
    <a class="popup_link" onfocus='this.blur();' href="javascript:showTestDetail('div_ft1.8')" >
        error</a>

    <div id='div_ft1.8' class="popup_window">
        <div style='text-align: right; color:red;cursor:pointer'>
        <a onfocus='this.blur();' onclick="document.getElementById('div_ft1.8').style.display = 'none' " >
           [x]</a>
        </div>
        <pre>
        
ft1.8: Traceback (most recent call last):
  File "/Users/jason.lik/DaDa_project/abc-qa-app-UITest/common/operate_element.py", line 186, in main_operate
    driver.implicitly_wait(20)
  File "/Users/jason.lik/DaDa_project/abc-qa-app-UITest/venv/lib/python3.6/site-packages/selenium/webdriver/remote/webdriver.py", line 912, in implicitly_wait
    'implicit': int(float(time_to_wait) * 1000)})
  File "/Users/jason.lik/DaDa_project/abc-qa-app-UITest/venv/lib/python3.6/site-packages/selenium/webdriver/remote/webdriver.py", line 321, in execute
    self.error_handler.check_response(response)
  File "/Users/jason.lik/DaDa_project/abc-qa-app-UITest/venv/lib/python3.6/site-packages/appium/webdriver/errorhandler.py", line 29, in check_response
    raise wde
  File "/Users/jason.lik/DaDa_project/abc-qa-app-UITest/venv/lib/python3.6/site-packages/appium/webdriver/errorhandler.py", line 24, in check_response
    super(MobileErrorHandler, self).check_response(response)
  File "/Users/jason.lik/DaDa_project/abc-qa-app-UITest/venv/lib/python3.6/site-packages/selenium/webdriver/remote/errorhandler.py", line 242, in check_response
    raise exception_class(message, screen, stacktrace)
selenium.common.exceptions.InvalidSessionIdException: Message: A session is either terminated or not started


During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/Users/jason.lik/DaDa_project/abc-qa-app-UITest/common/operate_element.py", line 152, in wrapper
    result = function(*a, **w)
  File "/Users/jason.lik/DaDa_project/abc-qa-app-UITest/testIOS/test_student_client.py", line 61, in test08_enter_oral_practice
    enter_oral_practice(self.driver)
  File "/Users/jason.lik/DaDa_project/abc-qa-app-UITest/testIOS/student_client/case/test08_oral_practice.py", line 17, in enter_oral_practice
    __case_collection(driver, 'ipad')
  File "/Users/jason.lik/DaDa_project/abc-qa-app-UITest/testIOS/student_client/case/test08_oral_practice.py", line 12, in __case_collection
    main_operate(driver, platform, **oral_practice)
  File "/Users/jason.lik/DaDa_project/abc-qa-app-UITest/common/operate_element.py", line 207, in main_operate
    yaml + platform + '/error_screenshot/') + '/' + get_current_hour_minute() + '_error.png')
  File "/Users/jason.lik/DaDa_project/abc-qa-app-UITest/common/operate_element.py", line 112, in __error_screenshot
    driver.get_screenshot_as_file(screenshot_path)
  File "/Users/jason.lik/DaDa_project/abc-qa-app-UITest/venv/lib/python3.6/site-packages/selenium/webdriver/remote/webdriver.py", line 1032, in get_screenshot_as_file
    png = self.get_screenshot_as_png()
  File "/Users/jason.lik/DaDa_project/abc-qa-app-UITest/venv/lib/python3.6/site-packages/selenium/webdriver/remote/webdriver.py", line 1064, in get_screenshot_as_png
    return base64.b64decode(self.get_screenshot_as_base64().encode('ascii'))
  File "/Users/jason.lik/DaDa_project/abc-qa-app-UITest/venv/lib/python3.6/site-packages/selenium/webdriver/remote/webdriver.py", line 1074, in get_screenshot_as_base64
    return self.execute(Command.SCREENSHOT)['value']
  File "/Users/jason.lik/DaDa_project/abc-qa-app-UITest/venv/lib/python3.6/site-packages/selenium/webdriver/remote/webdriver.py", line 321, in execute
    self.error_handler.check_response(response)
  File "/Users/jason.lik/DaDa_project/abc-qa-app-UITest/venv/lib/python3.6/site-packages/appium/webdriver/errorhandler.py", line 29, in check_response
    raise wde
  File "/Users/jason.lik/DaDa_project/abc-qa-app-UITest/venv/lib/python3.6/site-packages/appium/webdriver/errorhandler.py", line 24, in check_response
    super(MobileErrorHandler, self).check_response(response)
  File "/Users/jason.lik/DaDa_project/abc-qa-app-UITest/venv/lib/python3.6/site-packages/selenium/webdriver/remote/errorhandler.py", line 242, in check_response
    raise exception_class(message, screen, stacktrace)
selenium.common.exceptions.InvalidSessionIdException: Message: A session is either terminated or not started


During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/Users/jason.lik/DaDa_project/abc-qa-app-UITest/common/operate_element.py", line 158, in wrapper
    t(*a)
TypeError: 'classmethod' object is not callable
Traceback (most recent call last):
  File "/Users/jason.lik/DaDa_project/abc-qa-app-UITest/common/operate_element.py", line 186, in main_operate
    driver.implicitly_wait(20)
  File "/Users/jason.lik/DaDa_project/abc-qa-app-UITest/venv/lib/python3.6/site-packages/selenium/webdriver/remote/webdriver.py", line 912, in implicitly_wait
    'implicit': int(float(time_to_wait) * 1000)})
  File "/Users/jason.lik/DaDa_project/abc-qa-app-UITest/venv/lib/python3.6/site-packages/selenium/webdriver/remote/webdriver.py", line 321, in execute
    self.error_handler.check_response(response)
  File "/Users/jason.lik/DaDa_project/abc-qa-app-UITest/venv/lib/python3.6/site-packages/appium/webdriver/errorhandler.py", line 29, in check_response
    raise wde
  File "/Users/jason.lik/DaDa_project/abc-qa-app-UITest/venv/lib/python3.6/site-packages/appium/webdriver/errorhandler.py", line 24, in check_response
    super(MobileErrorHandler, self).check_response(response)
  File "/Users/jason.lik/DaDa_project/abc-qa-app-UITest/venv/lib/python3.6/site-packages/selenium/webdriver/remote/errorhandler.py", line 242, in check_response
    raise exception_class(message, screen, stacktrace)
selenium.common.exceptions.InvalidSessionIdException: Message: A session is either terminated or not started


During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/Users/jason.lik/DaDa_project/abc-qa-app-UITest/common/operate_element.py", line 152, in wrapper
    result = function(*a, **w)
  File "/Users/jason.lik/DaDa_project/abc-qa-app-UITest/testIOS/test_student_client.py", line 61, in test08_enter_oral_practice
    enter_oral_practice(self.driver)
  File "/Users/jason.lik/DaDa_project/abc-qa-app-UITest/testIOS/student_client/case/test08_oral_practice.py", line 17, in enter_oral_practice
    __case_collection(driver, 'ipad')
  File "/Users/jason.lik/DaDa_project/abc-qa-app-UITest/testIOS/student_client/case/test08_oral_practice.py", line 12, in __case_collection
    main_operate(driver, platform, **oral_practice)
  File "/Users/jason.lik/DaDa_project/abc-qa-app-UITest/common/operate_element.py", line 207, in main_operate
    yaml + platform + '/error_screenshot/') + '/' + get_current_hour_minute() + '_error.png')
  File "/Users/jason.lik/DaDa_project/abc-qa-app-UITest/common/operate_element.py", line 112, in __error_screenshot
    driver.get_screenshot_as_file(screenshot_path)
  File "/Users/jason.lik/DaDa_project/abc-qa-app-UITest/venv/lib/python3.6/site-packages/selenium/webdriver/remote/webdriver.py", line 1032, in get_screenshot_as_file
    png = self.get_screenshot_as_png()
  File "/Users/jason.lik/DaDa_project/abc-qa-app-UITest/venv/lib/python3.6/site-packages/selenium/webdriver/remote/webdriver.py", line 1064, in get_screenshot_as_png
    return base64.b64decode(self.get_screenshot_as_base64().encode('ascii'))
  File "/Users/jason.lik/DaDa_project/abc-qa-app-UITest/venv/lib/python3.6/site-packages/selenium/webdriver/remote/webdriver.py", line 1074, in get_screenshot_as_base64
    return self.execute(Command.SCREENSHOT)['value']
  File "/Users/jason.lik/DaDa_project/abc-qa-app-UITest/venv/lib/python3.6/site-packages/selenium/webdriver/remote/webdriver.py", line 321, in execute
    self.error_handler.check_response(response)
  File "/Users/jason.lik/DaDa_project/abc-qa-app-UITest/venv/lib/python3.6/site-packages/appium/webdriver/errorhandler.py", line 29, in check_response
    raise wde
  File "/Users/jason.lik/DaDa_project/abc-qa-app-UITest/venv/lib/python3.6/site-packages/appium/webdriver/errorhandler.py", line 24, in check_response
    super(MobileErrorHandler, self).check_response(response)
  File "/Users/jason.lik/DaDa_project/abc-qa-app-UITest/venv/lib/python3.6/site-packages/selenium/webdriver/remote/errorhandler.py", line 242, in check_response
    raise exception_class(message, screen, stacktrace)
selenium.common.exceptions.InvalidSessionIdException: Message: A session is either terminated or not started


During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/Users/jason.lik/DaDa_project/abc-qa-app-UITest/common/operate_element.py", line 158, in wrapper
    t(*a)
TypeError: 'classmethod' object is not callable


        </pre>
    </div>
    <!--css div popup end-->

    </td>
</tr>

<tr id='ft1.9' class='none'>
    <td class='errorCase'><div class='testcase'>test09_enter_baibao_box</div></td>
    <td colspan='5' align='center'>

    <!--css div popup start-->
    <a class="popup_link" onfocus='this.blur();' href="javascript:showTestDetail('div_ft1.9')" >
        error</a>

    <div id='div_ft1.9' class="popup_window">
        <div style='text-align: right; color:red;cursor:pointer'>
        <a onfocus='this.blur();' onclick="document.getElementById('div_ft1.9').style.display = 'none' " >
           [x]</a>
        </div>
        <pre>
        
ft1.9: Traceback (most recent call last):
  File "/Users/jason.lik/DaDa_project/abc-qa-app-UITest/common/operate_element.py", line 186, in main_operate
    driver.implicitly_wait(20)
  File "/Users/jason.lik/DaDa_project/abc-qa-app-UITest/venv/lib/python3.6/site-packages/selenium/webdriver/remote/webdriver.py", line 912, in implicitly_wait
    'implicit': int(float(time_to_wait) * 1000)})
  File "/Users/jason.lik/DaDa_project/abc-qa-app-UITest/venv/lib/python3.6/site-packages/selenium/webdriver/remote/webdriver.py", line 321, in execute
    self.error_handler.check_response(response)
  File "/Users/jason.lik/DaDa_project/abc-qa-app-UITest/venv/lib/python3.6/site-packages/appium/webdriver/errorhandler.py", line 29, in check_response
    raise wde
  File "/Users/jason.lik/DaDa_project/abc-qa-app-UITest/venv/lib/python3.6/site-packages/appium/webdriver/errorhandler.py", line 24, in check_response
    super(MobileErrorHandler, self).check_response(response)
  File "/Users/jason.lik/DaDa_project/abc-qa-app-UITest/venv/lib/python3.6/site-packages/selenium/webdriver/remote/errorhandler.py", line 242, in check_response
    raise exception_class(message, screen, stacktrace)
selenium.common.exceptions.InvalidSessionIdException: Message: A session is either terminated or not started


During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/Users/jason.lik/DaDa_project/abc-qa-app-UITest/common/operate_element.py", line 152, in wrapper
    result = function(*a, **w)
  File "/Users/jason.lik/DaDa_project/abc-qa-app-UITest/testIOS/test_student_client.py", line 65, in test09_enter_baibao_box
    enter_baibao_box(self.driver)
  File "/Users/jason.lik/DaDa_project/abc-qa-app-UITest/testIOS/student_client/case/test09_baibao_box.py", line 17, in enter_baibao_box
    __case_collection(driver, 'ipad')
  File "/Users/jason.lik/DaDa_project/abc-qa-app-UITest/testIOS/student_client/case/test09_baibao_box.py", line 12, in __case_collection
    main_operate(driver, platform, **baibao_box)
  File "/Users/jason.lik/DaDa_project/abc-qa-app-UITest/common/operate_element.py", line 207, in main_operate
    yaml + platform + '/error_screenshot/') + '/' + get_current_hour_minute() + '_error.png')
  File "/Users/jason.lik/DaDa_project/abc-qa-app-UITest/common/operate_element.py", line 112, in __error_screenshot
    driver.get_screenshot_as_file(screenshot_path)
  File "/Users/jason.lik/DaDa_project/abc-qa-app-UITest/venv/lib/python3.6/site-packages/selenium/webdriver/remote/webdriver.py", line 1032, in get_screenshot_as_file
    png = self.get_screenshot_as_png()
  File "/Users/jason.lik/DaDa_project/abc-qa-app-UITest/venv/lib/python3.6/site-packages/selenium/webdriver/remote/webdriver.py", line 1064, in get_screenshot_as_png
    return base64.b64decode(self.get_screenshot_as_base64().encode('ascii'))
  File "/Users/jason.lik/DaDa_project/abc-qa-app-UITest/venv/lib/python3.6/site-packages/selenium/webdriver/remote/webdriver.py", line 1074, in get_screenshot_as_base64
    return self.execute(Command.SCREENSHOT)['value']
  File "/Users/jason.lik/DaDa_project/abc-qa-app-UITest/venv/lib/python3.6/site-packages/selenium/webdriver/remote/webdriver.py", line 321, in execute
    self.error_handler.check_response(response)
  File "/Users/jason.lik/DaDa_project/abc-qa-app-UITest/venv/lib/python3.6/site-packages/appium/webdriver/errorhandler.py", line 29, in check_response
    raise wde
  File "/Users/jason.lik/DaDa_project/abc-qa-app-UITest/venv/lib/python3.6/site-packages/appium/webdriver/errorhandler.py", line 24, in check_response
    super(MobileErrorHandler, self).check_response(response)
  File "/Users/jason.lik/DaDa_project/abc-qa-app-UITest/venv/lib/python3.6/site-packages/selenium/webdriver/remote/errorhandler.py", line 242, in check_response
    raise exception_class(message, screen, stacktrace)
selenium.common.exceptions.InvalidSessionIdException: Message: A session is either terminated or not started


During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/Users/jason.lik/DaDa_project/abc-qa-app-UITest/common/operate_element.py", line 158, in wrapper
    t(*a)
TypeError: 'classmethod' object is not callable
Traceback (most recent call last):
  File "/Users/jason.lik/DaDa_project/abc-qa-app-UITest/common/operate_element.py", line 186, in main_operate
    driver.implicitly_wait(20)
  File "/Users/jason.lik/DaDa_project/abc-qa-app-UITest/venv/lib/python3.6/site-packages/selenium/webdriver/remote/webdriver.py", line 912, in implicitly_wait
    'implicit': int(float(time_to_wait) * 1000)})
  File "/Users/jason.lik/DaDa_project/abc-qa-app-UITest/venv/lib/python3.6/site-packages/selenium/webdriver/remote/webdriver.py", line 321, in execute
    self.error_handler.check_response(response)
  File "/Users/jason.lik/DaDa_project/abc-qa-app-UITest/venv/lib/python3.6/site-packages/appium/webdriver/errorhandler.py", line 29, in check_response
    raise wde
  File "/Users/jason.lik/DaDa_project/abc-qa-app-UITest/venv/lib/python3.6/site-packages/appium/webdriver/errorhandler.py", line 24, in check_response
    super(MobileErrorHandler, self).check_response(response)
  File "/Users/jason.lik/DaDa_project/abc-qa-app-UITest/venv/lib/python3.6/site-packages/selenium/webdriver/remote/errorhandler.py", line 242, in check_response
    raise exception_class(message, screen, stacktrace)
selenium.common.exceptions.InvalidSessionIdException: Message: A session is either terminated or not started


During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/Users/jason.lik/DaDa_project/abc-qa-app-UITest/common/operate_element.py", line 152, in wrapper
    result = function(*a, **w)
  File "/Users/jason.lik/DaDa_project/abc-qa-app-UITest/testIOS/test_student_client.py", line 65, in test09_enter_baibao_box
    enter_baibao_box(self.driver)
  File "/Users/jason.lik/DaDa_project/abc-qa-app-UITest/testIOS/student_client/case/test09_baibao_box.py", line 17, in enter_baibao_box
    __case_collection(driver, 'ipad')
  File "/Users/jason.lik/DaDa_project/abc-qa-app-UITest/testIOS/student_client/case/test09_baibao_box.py", line 12, in __case_collection
    main_operate(driver, platform, **baibao_box)
  File "/Users/jason.lik/DaDa_project/abc-qa-app-UITest/common/operate_element.py", line 207, in main_operate
    yaml + platform + '/error_screenshot/') + '/' + get_current_hour_minute() + '_error.png')
  File "/Users/jason.lik/DaDa_project/abc-qa-app-UITest/common/operate_element.py", line 112, in __error_screenshot
    driver.get_screenshot_as_file(screenshot_path)
  File "/Users/jason.lik/DaDa_project/abc-qa-app-UITest/venv/lib/python3.6/site-packages/selenium/webdriver/remote/webdriver.py", line 1032, in get_screenshot_as_file
    png = self.get_screenshot_as_png()
  File "/Users/jason.lik/DaDa_project/abc-qa-app-UITest/venv/lib/python3.6/site-packages/selenium/webdriver/remote/webdriver.py", line 1064, in get_screenshot_as_png
    return base64.b64decode(self.get_screenshot_as_base64().encode('ascii'))
  File "/Users/jason.lik/DaDa_project/abc-qa-app-UITest/venv/lib/python3.6/site-packages/selenium/webdriver/remote/webdriver.py", line 1074, in get_screenshot_as_base64
    return self.execute(Command.SCREENSHOT)['value']
  File "/Users/jason.lik/DaDa_project/abc-qa-app-UITest/venv/lib/python3.6/site-packages/selenium/webdriver/remote/webdriver.py", line 321, in execute
    self.error_handler.check_response(response)
  File "/Users/jason.lik/DaDa_project/abc-qa-app-UITest/venv/lib/python3.6/site-packages/appium/webdriver/errorhandler.py", line 29, in check_response
    raise wde
  File "/Users/jason.lik/DaDa_project/abc-qa-app-UITest/venv/lib/python3.6/site-packages/appium/webdriver/errorhandler.py", line 24, in check_response
    super(MobileErrorHandler, self).check_response(response)
  File "/Users/jason.lik/DaDa_project/abc-qa-app-UITest/venv/lib/python3.6/site-packages/selenium/webdriver/remote/errorhandler.py", line 242, in check_response
    raise exception_class(message, screen, stacktrace)
selenium.common.exceptions.InvalidSessionIdException: Message: A session is either terminated or not started


During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/Users/jason.lik/DaDa_project/abc-qa-app-UITest/common/operate_element.py", line 158, in wrapper
    t(*a)
TypeError: 'classmethod' object is not callable


        </pre>
    </div>
    <!--css div popup end-->

    </td>
</tr>

<tr id='total_row'>
    <td>Total</td>
    <td>9</td>
    <td>4</td>
    <td>0</td>
    <td>5</td>
    <td>&nbsp;</td>
</tr>
</table>

<div id='ending'>&nbsp;</div>

</body>
</html>
